### ISSUE TITLE
Repository Layout Query

### ISSUE THREAD

**Issue by milkypostman** [davidmiller/pony-mode#53]
I am a maintainer of [melpa](https://github.com/milkypostman/melpa) and we want to include a full-featured version of this package but due to restriction in how we currently package things, we have some restrictions on how files should be organized.  The main reason is to make it easier for people to create new packages rather than making the packaging more complex.

So I just am writing to see what the reason for putting your package files in the `src/` directory is as opposed to putting them directly in the root of the project and/or putting `snippets/` under the `src/` directory.

Basically, the package creation process looks for a common subdirectory path for all files in the package but it breaks in this instance.

I have added the package but it doesn't currently include the snippets which I would definitely like to add.  Anyways, seeing if you had opinion or input.  Not so much how we could change our end---cause we know how to do that---but rather to maybe see your reaction to that possibility.
